tests/queries/0_stateless/02536_replace_with_nonconst_needle_and_replacement.reference

```diff
@@ -139,10 +139,22 @@ Hexxo Worxd
 Hello World
 Hexlo World
 Hello World
+Hexxo Worxd
+Hello World
+Hexlo World
+Hello World
+Hello World
+Hello World
```

Comment:
Can we add some lines like `- non-const needle, non-const replacement` in this test?
Otherwise it's difficult to see which lines in the reference correspond to which lines in the test.